Android/AndroidEasingFunctions/cgmanifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Registrations":[ 
+    {
+      "Component": {
+        "Type": "Maven",
+        "Maven": {
+          "ArtifactId": "library",
+          "GroupId": "com.daimajia.easing",
+          "Version": "2.1"
```

Comment:
NuGetId ?

Reply:
Is this inside the Maven or the Component?